src/js/view/view.js

```diff
@@ -840,10 +834,6 @@ define([
                 _resizeMediaTimeout = setTimeout(_resizeMedia, 250);
             }
 
-            if (_model.get('aspectratio')) {
```

Comment:
We should also check  `!_model.get('renderCaptionsNatively')` so we don't resize the captions container unnecessarily.